server_admin/topics/authentication/x509.adoc

```diff
@@ -76,6 +76,7 @@ The certificate identity mapping can map the extracted user identity to an exist
 
 The following sections describe how to configure {appserver_name}/Undertow and the {project_name} Server to enable X.509 client certificate authentication.
 
+ifeval::["{kc_dist}" == "wildfly"]
```

Comment:
We have `https-client-auth`. Do we need an example for Quarkus or that is already covered by TLS guide?

Reply:
That's all configured by the new guides, which is the correct place